lib/persistence/relations/articles.rb

```diff
@@ -0,0 +1,15 @@
+module Persistence
+  module Relations
```

Comment:
I think `rom` allow you to specify the name you want. Here in the project the rom `auto registration` process is set here https://github.com/dry-rb/dry-web-blog/blob/master/system/boot/rom.rb#L34

But you can pass multiple options to modify it the way you need it https://github.com/rom-rb/rom/blob/master/core/spec/unit/rom/setup/auto_registration_spec.rb#L62